src/record/list.js

```diff
@@ -172,10 +172,13 @@ List.prototype.subscribe = function() {
 	}
 
 	//Make sure the callback is invoked with an empty array for new records
-	parameters.callback = function( callback ) {
+	var listCallback = function( callback ) {
 		callback( this.getEntries() );
 	}.bind( this, parameters.callback );
 
+	parameters.callback.wrappedCallback = listCallback;
+	parameters.callback = listCallback;
```

Comment:
Completely agree this is wierd. However we can't modify the record since it is a singleton.